README.md

```diff
@@ -128,7 +128,7 @@ receivers:
     address: "localhost:55678"
 
   zipkin:
-    address: "localhost:9411"
```

Comment:
This depends: if the collector then port is correct, if agent address is correct... we have an issue open to fix that.